core-web/libs/data-access/src/lib/dot-site/dot-site.service.ts

```diff
@@ -17,21 +17,20 @@ export interface SiteParams {
     providedIn: 'root'
 })
 export class DotSiteService {
-    private BASE_SITE_URL = '/api/v1/site';
-    private params: SiteParams = {
+    #BASE_SITE_URL = '/api/v1/site';
```

Comment:
Can we use a constant for this? to share the constant in the service and test, if the endpoint changes, the test will fail.

Reply:
@oidacra done!